frontend/packages/policy-editor/src/components/ExpandablePolicyCard/ResourceNarrowingList/ResourceNarrowingList.tsx

```diff
@@ -129,9 +122,7 @@ export const ResourceNarrowingList = ({
             size='small'
             dashedBorder
             variant='outline'
-            icon={
-              <PlusIcon title={t('policy_editor.narrowing_list_add_button')} fontSize='1.5rem' />
-            }
+            icon={<PlusIcon fontSize='1.5rem' />}
```

Comment:
Jeg ser at dette var slik fra før, men i utgangspunktet vil vi separere stil fra funksjonell kode så langt det lar seg gjøre, altså referere til CSS-klasser i stedet for å sette på `font-size` direkte.